heksher/app.py

```diff
@@ -0,0 +1,65 @@
+from asyncio import wait_for
+from logging import getLogger, INFO
+
+from databases import Database
+from envolved import EnvVar, Schema
+from envolved.parsers import CollectionParser
+from fastapi import FastAPI
+
+from heksher.db_logic import DBLogic
+
+connection_string = EnvVar('HEKSHER_DB_CONNECTION_STRING', type=str)
+startup_context_features = EnvVar('HEKSHER_STARTUP_CONTEXT_FEATURES', type=CollectionParser(';', str))
+
+
+class LogstashSettingSchema(Schema):
+    host: str = EnvVar()
+    port: int = EnvVar()
+
+
+logstash_settings = EnvVar('HEKSHER_LOGSTASH_', default=None, type=LogstashSettingSchema)
+
+
+class HeksherApp(FastAPI):
+    """
+    The application class
+    """
+    db: Database
+    db_logic: DBLogic
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    async def startup(self):
+        settings = logstash_settings.get()
+        if settings is not None:
+            try:
```

Comment:
heksher comes as an image, so I wouldn't work on that extra too much.. it's a little dependency.